LicenseOwners/Pages/Index.cshtml.cs

```diff
@@ -13,20 +13,26 @@ public class IndexModel : PageModel
         public void OnGet()
         {
             List<BusinessLicenseOwners> businessLicenseOwnersList = new List<BusinessLicenseOwners>();
-            BusinessLicenseOwners businessLicenseOwner = null;
-           
-                BusinessLicenses[] businessLicenses = BusinessLicenses.FromJson(getJSONData("https://data.cityofchicago.org/resource/r5kz-chrr.json?$where=account_number%20<%2051"));
-                ViewData["BusinessLicenses"] = businessLicenses;
 
-                BusinessOwners[] businessOwners = BusinessOwners.FromJson(getJSONData("https://data.cityofchicago.org/resource/ezma-pppn.json?$where=account_number%20<%2051"));
-                ViewData["BusinessOwners"] = businessOwners;
-           
+            BusinessLicenses[] businessLicenses = BusinessLicenses.FromJson(getJSONData("https://data.cityofchicago.org/resource/r5kz-chrr.json?$where=account_number%20<%2051"));
+            ViewData["BusinessLicenses"] = businessLicenses;
 
-            IDictionary<string, BusinessLicenses> licenseDictionary = new Dictionary<string, BusinessLicenses>();
+            BusinessOwners[] businessOwners = BusinessOwners.FromJson(getJSONData("https://data.cityofchicago.org/resource/ezma-pppn.json?$where=account_number%20<%2051"));
+            ViewData["BusinessOwners"] = businessOwners;
+
+            LicenseDetails(businessLicenses, businessOwners, businessLicenseOwnersList);
 
+        }
+        public void LicenseDetails(BusinessLicenses[] businessLicenses, BusinessOwners[] businessOwners, List<BusinessLicenseOwners> businessLicenseOwnersList)
+        {
+            BusinessLicenseOwners businessLicenseOwner = null;
+            IDictionary<string, BusinessLicenses> licenseDictionary = new Dictionary<string, BusinessLicenses>();
+            
             foreach (BusinessLicenses businessLicense in businessLicenses)
             {
-                if (!licenseDictionary.ContainsKey((businessLicense.AccountNumber+"_" +businessLicense.SiteNumber) + "")){
+                if (!licenseDictionary.ContainsKey((businessLicense.AccountNumber+"_" +businessLicense.SiteNumber) + ""))
```

Comment:
stay consitent on curlies.  Either follow the if test or put it on the line after... but stay consistent.